geniza/corpus/tests/test_import_data.py

```diff
@@ -821,6 +823,69 @@ def test_update_document_id_sequence():
 ]
 
 
+@pytest.mark.django_db
+def test_parse_notes():
+    # Ensure old_pgpids are parsed correctly
+    doc = Document.objects.create()
+    row = AttrMap({"notes": "Old PGPID: 6160", "tech_notes": ""})
+    import_data.Command().parse_notes(doc, row)
+    assert doc.old_pgpids == [6160]
+
+    doc = Document.objects.create()
+    row = AttrMap({"notes": "Old PGPIDs: 6160, 3600", "tech_notes": ""})
+    import_data.Command().parse_notes(doc, row)
+    assert 6160 in doc.old_pgpids and 3600 in doc.old_pgpids
+
+    doc = Document.objects.create()
+    row = AttrMap({"notes": "Old PGPIDs: 6160, 3600", "tech_notes": ""})
+    import_data.Command().parse_notes(doc, row)
+    assert 6160 in doc.old_pgpids and 3600 in doc.old_pgpids
+
+    doc = Document.objects.create()
+    row = AttrMap({"notes": "India; Old PGPID: 9242", "tech_notes": ""})
+    import_data.Command().parse_notes(doc, row)
+    assert 9242 in doc.old_pgpids
+
+    # Make sure ignored notes aren't included
+    doc = Document.objects.create()
+    row = AttrMap({"notes": "DISAGGREGATE", "tech_notes": ""})
+    import_data.Command().parse_notes(doc, row)
+    assert doc.notes == ""
```

Comment:
great to see all these use cases tested

please add test to confirm old pgpid string doesn't get included in note text